be/src/formats/orc/apache-orc/c++/src/ColumnReader.cc

```diff
@@ -1181,7 +1222,7 @@ void ListColumnReader::nextInternal(ColumnVectorBatch& rowBatch, uint64_t numVal
     }
 }
 
-void ListColumnReader::seekToRowGroup(std::unordered_map<uint64_t, PositionProvider>& positions) {
+void ListColumnReader::seekToRowGroup(PositionProviderMap& positions) {
```

Comment:
const PositionProviderMap&

Reply:
It will change content. I'll update with pointer version.